### PR TITLE
feat(MPM-553): add missing docblock

### PR DESCRIPTION
**Pull request by ruzakaranbol**

### CODE REVIEW

File: docker/dev/php/Dockerfile [hunk 1]
```diff
@@ -10,10 +10,7 @@ COPY files/php/ /phpIni
 
 # SYS: Install required packages
 RUN apk --no-cache upgrade && \
-    apk --no-cache add bash git sudo openssh autoconf gcc g++ make
-
-RUN apk add librdkafka librdkafka-dev \
-        --update-cache --repository http://dl-3.alpinelinux.org/alpine/edge/community
+    apk --no-cache add bash git sudo openssh autoconf gcc g++ make librdkafka librdkafka-dev
```
Comment (bajdzun) [inline docker/dev/php/Dockerfile:13]: While we are here, can we also bump alpine to 3.16?

Comment (nick-zh) [reply docker/dev/php/Dockerfile:13]: or even go for `3.17` :rocket:

Comment (bajdzun) [reply docker/dev/php/Dockerfile:13]: Now, yes 😄 Thanks man! 💪 

EDIT: There is no image for PHP 7.4 😄

Comment (nick-zh) [reply docker/dev/php/Dockerfile:13]: Ah yes sry, if you still have `7.4` as base, then that might not be possible. Keep in mind that `7.4` reached [EOL](https://www.php.net/supported-versions.php) though :v:

**Review by bajdzun — APPROVED** [review]

**Pull request merged**